### DISCUSSION TITLE
Lagoon on Minikube?

### DISCUSSION THREAD

**Discussion by socketwench** [uselagoon/lagoon#3831] [Q&A]
Is there a way to run Lagoon locally? Say, on Minikube? It looks possible, but I'm not sure how to set up the storage provider.

**Comment by shreddedbacon** [comment]
Yes, but we have ways to run lagoon locally for testing already https://docs.lagoon.sh/contributing-to-lagoon/developing-lagoon/#deploy-a-local-lagoon-development-stack-without-test-suites

Do you have a reason to use minikube specifically?
